source/configuration_manual/mail_location/dbox/index.rst

```diff
@@ -0,0 +1,134 @@
+  mail_location = sdbox:~/dbox
+
+For backwards compatibility, ``dbox`` is an alias to ``sdbox`` in the mail
+location. (This usage is deprecated.)
+
+To use **multi-dbox**, use the tag ``mdbox`` in the
+:ref:`mail location <mail_location_settings>`:
+
+.. code-block:: none
+
+  # multi-dbox
+  mail_location = mdbox:~/mdbox
+
+.. _dbox_settings_alt_storage:
+
+Alternate Storage
+^^^^^^^^^^^^^^^^^
+
+See :ref:`dbox alternate storage <dbox_mbox_format_alt_storage>` for further
+information.
+
+To specify an alternate storage area, use the ``ALT`` parameter in the mail
+location. For example:
+
+.. code-block:: none
+
+  mail_location = mdbox:/var/vmail/%d/%n:ALT=/altstorage/vmail/%d/%n
+
+will make Dovecot look for message data first under ``/var/vmail/%d/%n``
+("primary storage"), and if it is not found there it will look under
+``/altstorage/vmail/%d/%n`` ("alternate storage") instead. There's no problem
+having the same (identical) file in both storages.
+
+Keep the unmounted ``/altstorage`` directory permissions such that Dovecot
+mail processes can't create directories under it (e.g. ``root:root 0755``).
+This way if the alt storage isn't mounted for some reason, Dovecot won't
+think that all the messages in alt storage were deleted and lose their flags.
+
+.. _dbox_settings_mailbox_directory_name:
+
+Mailbox Directory Name
+^^^^^^^^^^^^^^^^^^^^^^
+
+When using the default hierarchical layout, there is a potential for naming
+collisions between dbox's ``dbox-Mails/`` subdirectory and mail folders of
+the same name.
+
+For example, consider a mail folder "foo/bar". Under the default hierarchical layout, data about this mail folder would be stored at {{{~/mdbox/mailboxes/foo/bar/dbox-Mails/}}}.
+
+For example, consider a mail folder ``foo/bar``. Under the default
+hierarchical layout, data for this mail folder will be stored at
+``~/mdbox/mailboxes/foo/bar/dbox-Mails/``. If the user then tries to create a
+mail folder ``foo/bar/dbox-Mails``, this would then imply that data should be
+stored at ``~/mdbox/mailboxes/foo/bar/dbox-Mails/dbox-Mails/``. But this
+would overlap the ``dbox-Mails/`` subdirectory of mail folder ``foo/bar``.
+
+This may not be a problem in many installations, but if a risk of collisions
+with the default name ``dbox-Mails`` is perceived, then the ``DIRNAME``
+parameter can be used. For example, if we specify mail location as:
+
+.. code-block:: none
+
+  mail_location = mdbox:~/mdbox:DIRNAME=DbOx-mAiLs
+
+then this will make data for mail folders be stored in a subdirectory
+``DbOx-mAiLs/`` instead of the default ``dbox-Mails/``, so a mail folder
+``foo/bar`` would have data stored at
+``~/mdbox/mailboxes/foo/bar/DbOx-mAiLs/``.
+
+The value for ``DIRNAME`` should be chosen carefully so as to minimise the chances of clashing with mail folder names. In the example here, unusual upper/lower casing has been used.
+
+mdbox Configuration Settings
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. _setting-mdbox_preallocate_space:
```

Comment:
These conflict with `settings/core.rst`. Also these should be mentioned there, not here.

Reply:
See above: settings don't have to live on core settings page and, if not, then the settings link should go to the setting itself, not a stub entry with yet another link.